bool robot_move_commanded